### PR TITLE
Separate out the deployment config for parent user invitation

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
### Purpose
- $subject to keep the fine grain access to handle the feature properties.

### CODE REVIEW

**Review by NipuniBhagya — APPROVED** [review]

**Review by pavinduLakshan — APPROVED** [review]

File: apps/console/src/public/deployment.config.json
Comment (pavinduLakshan) [inline apps/console/src/public/deployment.config.json:1]: If this configuration is required for SaaS deployment, you need to send PR to cloud extensions as well.

File: apps/console/src/public/deployment.config.json [hunk 1]
```diff
@@ -608,6 +608,25 @@
                     ]
                 }
             },
+            "parentUserInvitation": {
```
Comment (brionmario) [inline apps/console/src/public/deployment.config.json:611]: Add a `j2` mapping as well.

Comment (brionmario) [reply apps/console/src/public/deployment.config.json:611]: And also update the `default.json` in `carbon-identity-framework`

Comment (ShanChathusanda93) [reply apps/console/src/public/deployment.config.json:611]: `J2 ` mapping updated and the `default.json` in `carbon-identity-framework` is updated with https://github.com/wso2/carbon-identity-framework/pull/5672

**Review by brionmario — APPROVED** [review]

**Pull request merged**